modules/gapi/test/s11n/gapi_s11n_tests.cpp

```diff
@@ -511,4 +533,18 @@ TEST_F(S11N_Basic, Test_Custom_Type) {
     MyCustomType new_var = cv::gapi::s11n::detail::S11N<MyCustomType>::deserialize(is);
     EXPECT_EQ(var, new_var);
 }
+
+TEST_F(S11N_Basic, Test_Custom_CompileArg) {
+    MyCustomType customVar{1248, "World", {1280, 720, 640, 480}, {{32434142342, 5}, {7, 34242432}}};
+    int intVar{3};
+    GCompileArgs args = cv::compile_args(customVar, intVar);
+
+    std::vector<char> sArgs = cv::gapi::serialize(args);
+    GCompileArgs dArgs = cv::gapi::deserialize<GCompileArgs, MyCustomType, int>(sArgs);
```

Comment:
Why passing int? Is `int` a compile arg here? I'd avoid stuff like this.

Reply:
It is removed.